### DISCUSSION TITLE
Cure Model for Prediction of Testdata with unknown Cure-Rate Distribution

### DISCUSSION THREAD

**Discussion by AndreasHechtel** [CamDavidsonPilon/lifelines#1294] [Q&A]
Hi @CamDavidsonPilon,

i am currently trying to come up with a model for my company to predict survival curves for customer vouchers to be discharged over a given periode of time after being sold. Tough, as there is an empirical chance that vouchers can expire rather than being discharged, the Cure Model that you suggest seems quite interesting to me!

Since a voucher, that has been expired, has a 0% chance to be discharged afterwards, in my thinking, it comes pretty close to the idea of being "cured".

Problem now is, that i also need to predict for active customer vouchers, where, just like my target variable (Discharge at T(x)), i still lack of information about the expiration event taking place or not, as these vouchers maybe have just been sold and therefore are only at the beginnen of their "survival" journey.

Did you ever come across similar problems and have some advice, how i could deal with that? :)

Thanks alot and greetings
Andreas

**Comment by CamDavidsonPilon** [comment]
Hi @AndreasHechtel, 

I don't think the problem is a "cured" model problem - I actually think you have _competing risks_. The expiration is risk that will prevent your vouchers from being discharged. A similar situation is studying time-to-heart-disease, but having other causes of death as competing risks (since dead ppl can't develop heart disease). I would suggest doing some reading on competing risks first, to see if your problem matches that.

**Reply by AndreasHechtel** [reply]
Hi @CamDavidsonPilon,
thanks for the super quick response!
Competing risks seems intuitive in my regards, you are right.
But as far as my investigations go so far, there does not seem to be a parametric model with competing risk controll for prediction purpose available yet.
In result i am playing with the idea to use average values for fitting my test data when there are no empiric expiration event available yet and see how it goes.

Can you therefore give me your thoughts on what are the differences between "cure" and "competing risk"?

Thanks!
Andreas

**Reply by robchadil** [reply]
@AndreasHechtel this is a similar problem to what I am grappling with now. Did you ever find resources to implement parametric competing risks?
